nextflow_schema.json

```diff
@@ -107,12 +108,14 @@
                 "host_organism": {
                     "type": "string",
                     "description": "Change to custom name if desired, ie Human_hela_cells",
-                    "fa_icon": "far fa-file"
+                    "fa_icon": "far fa-file",
+                    "default": "host"
```

Comment:
Not quite - this will/can change to the specific host organism they want to use. This could be human cells, blood mouse etc etc.